### PR TITLE
Shrink REPL svds docs as well

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: stdlib/IterativeEigenSolvers/docs/src/index.md [hunk 1]
```diff
@@ -201,6 +201,54 @@ julia> λ
     | `nothing`       | ordinary (forward)               | ``Av = Bv\\lambda``                |
     | real or complex | inverse with level shift `sigma` | ``(A - \\sigma B )^{-1}B = v\\nu`` |
 
+`svds` computes the largest singular values `s` of `A` using implicitly restarted Lanczos
```
Comment (fredrikekre) [inline stdlib/IterativeEigenSolvers/docs/src/index.md]: Could add a markdown header here
```
## Singular-value decomposition
```

and one for the `eigs` up top too :)

**Comment by andreasnoack** [comment]
Are we really sure that it is an improvement to shrink the doc string? Personally, I'm not bothered by the lenght of the output and I'd prefer to have the details available there. The documentation attached to each function in other technical computing environments is typically quite extensive and e.g. unix man pages are also quite long and I'd think that most people see that as a good thing. Hence, I'd rather that other Julia docstring become more like `svds` than the opposite.

**Comment by ViralBShah** [comment]
Doesn't appear that we are removing any information here.

**Comment by Sacha0** [comment]
I sympathize with Andreas' position: Having complete documentation available at the REPL is lovely. Best! :)

**Comment by ViralBShah** [comment]
I agree, it is nice to have the full documentation.

**Pull request closed without merging**